src/manage.c

```diff
@@ -88,15 +90,46 @@ static acq_status_t acq_status[PLATFORM_SIGNAL_COUNT];
 #define DOPP_UNCERT_ALMANAC 4000
 #define DOPP_UNCERT_EPHEM   500
 
+#define COMPILER_BARRIER() asm volatile ("" : : : "memory")
+
+#define TRACKING_STARTUP_FIFO_SIZE 8
```

Comment:
@jacobmcnamee: I'd propose to add a comment that TRACKING_STARTUP_FIFO_SIZE must be power of 2 as otherwise the algorithm won't work correctly.